README.md

```diff
@@ -0,0 +1,6 @@
+# jira-swaggers
+
+This repository holds a stable version of [Atlassian JIRA Cloud platform](https://developer.atlassian.com/cloud/jira/platform/rest/v3/) and [Atlassian JIRA Software Cloud](https://developer.atlassian.com/cloud/jira/software/rest) API swagger definitions. 
+## Using this repository
+The "Fetch Swaggers" action is scheduled to run nightly, but can also be run manually via the "Actions" menu.
+When there is an updated swagger, it will automatically create a PR that should be reviewed and merged manually once it is considered "stable"
```

Comment:
```suggestion
This repository holds a stable version of [Atlassian JIRA Cloud platform](https://developer.atlassian.com/cloud/jira/platform/rest/v3/) and [Atlassian JIRA Software Cloud](https://developer.atlassian.com/cloud/jira/software/rest) API swagger definitions. 

## Using this repository
The "Fetch Swaggers" action is scheduled to run nightly, but can also be run manually via the "Actions" menu.

When there is an updated swagger, it will automatically create a PR that should be reviewed and merged manually once it is considered "stable"
```

without the extra space it looked bad when rendered